components/document360/sources/new-article-in-project-version/new-article-in-project-version.mjs

```diff
@@ -0,0 +1,68 @@
+import document360 from "../../document360.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "document360-new-article-in-project-version",
+  name: "New Article in Project Version",
+  description:
+    "Emit new event when a new article is created within a project version. [See the documentation](https://apidocs.document360.com/apidocs/project-version-articles)",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    document360,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+    projectVersionId: {
+      propDefinition: [
+        document360,
+        "projectVersionId",
+      ],
+    },
+  },
+  hooks: {
+    async deploy() {
+      await this.getAndProcessItems(10);
+    },
+  },
+  methods: {
+    _getSavedItems() {
+      return this.db.get("savedItems") ?? [];
+    },
+    _setSavedItems(value) {
+      this.db.set("savedItems", value);
+    },
+    async getAndProcessItems(maxItems = 0) {
+      const savedItems = this._getSavedItems();
+      const response = await this.document360.getArticles(
+        this.projectVersionId,
+      );
+      if (!response?.success) {
+        throw new Error(JSON.stringify(response));
+      }
+
+      response.data
+        ?.filter((item) => !savedItems.includes(item.id))
+        .forEach((article, index) => {
+          if (!maxItems || index < maxItems) {
+            this.$emit(article, {
+              id: article.id,
```

Comment:
The unique deduping only stores the last 50 or 100 items - since this endpoint is not limited to returning only a small amount of items, it could result in duplicate events beyond that amount